queries.sql

```diff
@@ -54,4 +48,40 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts)
 FROM animals
 WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
```

Comment:
The result of the corresponding screenshot to this query is not correct. Eg., *Gabumon* is missing from the list and so is *Pikachu* along with *Devimon*. Kindly fix this error.